Makefile

```diff
@@ -313,6 +313,10 @@ manifests: ## Generate K8s manifests e.g. CRD, RBAC etc.
 	mv ${TMPDIR}/cilium.io_ciliumnodes.yaml ./examples/crds/ciliumnodes.yaml
 	mv ${TMPDIR}/cilium.io_ciliumexternalworkloads.yaml ./examples/crds/ciliumexternalworkloads.yaml
 	mv ${TMPDIR}/cilium.io_ciliumlocalredirectpolicies.yaml ./examples/crds/ciliumlocalredirectpolicies.yaml
+	rm -rf $(TMPDIR)
+	$(eval TMPDIR := $(shell mktemp -d))
+	cd "./vendor/sigs.k8s.io/controller-tools/cmd/controller-gen" && \
+	go run ./... $(CRD_OPTIONS) paths="$(PWD)/pkg/k8s/apis/cilium.io/v2alpha1" output:crd:artifacts:config="$(TMPDIR)";
```

Comment:
I don't think this is strictly necessary. Couldn't we simply append the v2alpha1 path to the original `paths=` arg?

```
go run ./... $(CRD_OPTIONS) paths="$(PWD)/pkg/k8s/apis/cilium.io/v2,$(PWD)/pkg/k8s/apis/cilium.io/v2alpha1" output:crd:artifacts:config="$(TMPDIR)";  
```

Then we won't need to invoke the tool twice

Reply:
Good point. This is updated. Thx.